tests/e2e/singlecard/test_offline_inference.py

```diff
@@ -35,6 +35,7 @@
 MODELS = [
     "Qwen/Qwen2.5-0.5B-Instruct",
     "Qwen/Qwen3-0.6B-Base",
+    "Qwen/Qwen3-30B-A3B",
```

Comment:
This file is too huge and will cost lots of time to run ci, please try to used reduce layer model: https://vllm-ascend.readthedocs.io/en/latest/developer_guide/contribution/testing.html#e2e-test-example